tools/osbuilder/tests/test_config.sh

```diff
@@ -33,29 +33,3 @@ distro_in_set() {
 	done
 	return 1
 }
-
```

Comment:
Wrong commit... this hunk should go to `tools: Remove CI variable in test config osbuilder script`.

Reply:
done